cafebabel/static/css/styles.css

```diff
@@ -1685,15 +1685,15 @@ select#author {
   margin-bottom: 2rem;
 }
 
-.article-page article p.video-wrapper {
+p.video-wrapper {
```

Comment:
Why did you choose to take this tag out of the _article-page_ context? Is it used for the tag summary?

Reply:
it is.